modules/oadp-debugging-oc-cli.adoc

```diff
@@ -34,33 +34,33 @@ $ oc logs pod/<velero>
 [id="oc-velero-debug-logs_{context}"]
 == Velero pod debug logs
 
-Use the `oc edit` command to set the `Velero` pod logs to debug level:
+You can specify the Velero log level in the DataProtectionApplication resource as shown in the example below.
```

Comment:
```suggestion
You can specify the Velero log level in the `DataProtectionApplication` resource as shown in the following example.
```

> ! below adjective, adverb, preposition
Do not use to refer to an earlier software version or fix level. Use "earlier". Do not use to indicate a relative location in a document, as in "the information below". Use "the information later in this section". Alternatively, you can use "following" to refer to a table, to art, or to a list. Example: "The following table shows ..."

Please review the [markup reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference).

Reply:
@kalexand-rh 
Done